Makefile

```diff
@@ -10,27 +10,25 @@ PKGS := $(foreach pkg, $(sort $(dir $(SRCS))), $(pkg))
 TESTARGS ?=
 
 default:
-	GO15VENDOREXPERIMENT=1 go build -v
+	go build -v
```

Comment:
Why you've removed GO15VENDOREXPERIMENT? People with go 1.5 won't be able to build rocker and  there is nothing wrong using this variable with go1.6. It even't won't hurt us if we will use it with go1.7